src/worker/utils/molecule.ts

```diff
@@ -47,6 +47,26 @@ export const get_molecule = (smiles: string, RDKit: RDKitModule) => {
   }
 };
 
+export const get_query_molecule_memory_unsafe = (structure: string, RDKit: RDKitModule) => {
```

Comment:
Better not export this as it's memory_unsafe ( to avoid OOM)

Reply:
Removed export